app/__init__.py

```diff
@@ -292,6 +298,7 @@ def setup_babel(application):
     application.babel = Babel(application)
     application.babel.localeselector(get_locale)
     application.jinja_env.add_extension('jinja2.ext.i18n')
+    application.jinja_env.cache = {}
```

Comment:
im not too sure if its even effective here. but I'm not sure why it would behave any differently on Apache?